Seasons Defense/Assets/Scripts/TowerDecider.cs

```diff
@@ -0,0 +1,31 @@
+using UnityEngine;
+
+public class TowerDecider : MonoBehaviour
+{
+    public Tower leftTower;
+    public Tower middleTower;
+    public Tower rightTower;
+
+    private float _third = Screen.width / 3f;
+
+    private void Update()
+    {
+        
+        if(Input.GetMouseButtonDown(0))
+        {
+            float xPosOnClick = Input.mousePosition.x;
+
+            //TODO: Add more logic to this algorithm to decide the next best turret to use,
+            //      if the current one has no missiles left
+
+            if (xPosOnClick < _third) TowerToShoot(leftTower);
+            else if (xPosOnClick > _third && xPosOnClick < (_third * 2f)) TowerToShoot(middleTower);
+            else TowerToShoot(rightTower);
+        }
+    }
+
+    private void TowerToShoot(Tower tower)
+    {
+        tower.Fire();
+    }
```

Comment:
This will be redundant unless we need to do more than Fire a missile from the TowerDecider (I don't think so, particle effects and such can be done from the Missile script)